### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by efoosag**
- Using the URLs provided for Crypto Market Data
- Add fetch Data functionality from APIs
- Display data gotten from the Api
- Add Stylings
- Carry jest test
- Deployment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,46 +1,68 @@
...
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+👤 **Victor Osagie**
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+- GitHub: [@efoosag](https://github.com/efoosag)
+- Twitter: [@Victor Osagie](https://www.twitter.com/Victorosagie08)
+- LinkedIn: [@VictorOsagie](https://www.linkedin.com/in/victor-osagie-a713ba22b/)
 
-## Learn More
+## Acknowledgments
```
Comment (gemmen29) [inline README.md:62]: - [ ] Please add an Acknowledgements that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: src/components/CoinDetails.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { useLocation, Link } from 'react-router-dom';
+import { BsX } from 'react-icons/bs';
+
+const CoinDetails = () => {
+  const {
+    state: { crypto },
+  } = useLocation();
+  return (
+    <div className="container">
```
Comment (gemmen29) [inline src/components/CoinDetails.js:10]: - [ ] Please fix the horizontal scrolling on the details page.

https://user-images.githubusercontent.com/24703171/182432173-8338d2c4-3043-4a12-8916-c81b3ef74ce2.mp4

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @efoosag, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DammyShittu — APPROVED** [review]
Hi @efoosag,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**